addons/mail/static/src/core/common/message.js

```diff
@@ -133,9 +135,7 @@ export class Message extends Component {
         });
         useEffect(
             (editingMessage) => {
-                if (this.props.message.eq(editingMessage)) {
-                    messageActionsRegistry.get("edit").onClick(this);
-                }
+                this.state.isEditing = this.props.message.eq(editingMessage);
```

Comment:
Also kept for retro-compatibility, but as stated above, this local state should be removed.